_posts/2018-01-06-logs-for-the-Community-meeting-held-on-2018-01-06.md

```diff
@@ -0,0 +1,320 @@
+**\<ErCiccione>** nothing from me but if anybody has question, here to answer about localizations  
+**\<sgp>** I'll proceed then onto the next topic  
+**\<ordtrogen>** I have a comment on localization, when topic's up  
+**\<ordtrogen>** *the topic*  
+**\<sgp>** @ordtrogen you can ask it now  
+**\<ErCiccione>** sure, go ahead ordtrogen  
+**\<ordtrogen>** (pasting some pre-written text ...)  
+**\<ordtrogen>** I've only recently gotten more involved in translation/localization but pretty quickly I have discovered a couple of issues with the localizable strings, both in the CLI and GUI project. So I think it needs some polishing to look proper. (I am filing issues as we speak)  There also doesn't seem to exist much in the way of help documentation. I know there is some texts on getmonero.org, but I'd argue there should fir  
+comes with the installation, such as a man file for unix and maybe some other formats for Mac and Windows.  Which makes me think we should spend a little more time on the "foundations" of the texts we write. Like improving the UI English.  While @ercicciones work on the localization project obviously is fantastic, I'm thinking, what quality can the translations have unless the source (en) is correct?  
+**\<rehrar>** Agree wholeheartedly.  
+**\<rehrar>** As I work on the website here and there, most of the text is not written by me, except for the newer pages (and even that should be reviewed). Some of the copy does not read well, or is over flowery in some places, or other issues.  
+**\<ErCiccione>** yes, a lot of good points. the source file has been polished not a month ago, but would be great to have a clean and reviewed text to work on  
+**\<sgp>** I think most people agree documentation needs to be improved across the board  
+**\<ErCiccione>** about documentation  
+**\<ordtrogen>** I'm thinking a man file for monerod would be in order. Need not be too extensive  
+**\<ordtrogen>** (linux)  
+**\<ordtrogen>** And I'm not sure, what is displayed by $monerod --help    is not localizable, right?  
+**\<ErCiccione>** i'm working in creating a wiki with everything necessary for a translator, it can be found here https://taiga.getmonero.org/project/erciccione-monero-localization/wiki/home it need some improving, but following translators can take days some times, and i actually have no time, would be great to have some help on that side  
+**\<ErCiccione>** @ordtrogen it is actually, you can find the references to the code in the  .ts files  
+**\<ordtrogen>** ok. I just wanted to bring up the issue. As soon as I have to write something, I will.  
+**\<ErCiccione>** iirc a man file was partially done few days ago, but yes, there  
+**\<ordtrogen>** Will try to finish the testing and issue reporting first  
+**\<ErCiccione>** \* there is needing of a lot of clean documentation, thank you for bringing that up  
+**\<ordtrogen>** Oh, a man file is somewhere in the repos?  
+**\<ordtrogen>** CLI or GUI`?  
```

Comment:
Could this ` break the formatting of the blog post?

Reply:
I don't think so, the code block opened by the backtick doesn't carry on between lines.

##### Source
```
Like `

this.
```

##### Result
Like `

this.

Reply:
Alright!